widgets/common/src/inputs/utils/getInputField.tsx

```diff
@@ -184,6 +185,45 @@ export default function getInputField(
                 />
             );
         }
+        case 'capability_value': {
+            const fetchUrl = '/searches/capabilities';
+
+            // Formatting returned deployments to capabilities.
+            const itemsFormatter = (deployments: any) =>
```

Comment:
```suggestion
            const itemsFormatter = (deployments: any[]) =>
```

Just to indicate that it's indeed should be an array (as stated inside the [DynamicDropdown interface](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/widgets/common/src/components/DynamicDropdown.tsx#L94)) 😉

Reply:
Right! Commited.